app/src/main/res/values/strings.xml

```diff
@@ -76,22 +76,22 @@
     <!-- Dialogs -->
     <string name="button_dismiss">Dismiss</string>
     <string name="guide">Guide</string>
-    <string name="hold_on">Hold on!</string>
-    <string name="magisk_vanced">Looks like you\'re using the Magisk/TWRP version of Vanced, which is discontinued and can not be updated using this app. Please remove it first by removing the magisk module/using TWRP Vanced uninstaller.</string>
+    <string name="hold_on">Stop!</string>
```

Comment:
"Hold on!" Sounds better imo

Reply:
We don't need it to sounds better.
We need to get attention from users. Therefore, Stop, it is shorter but more compact. Hold on is kinda soft, this is an important notice.